lib/views/public_body/_more_info.html.erb

```diff
@@ -14,6 +14,13 @@
   <%= link_to _('Disclosure log'), public_body.disclosure_log %><br>
 <% end %>
 
+<% if public_body.has_tag?('statistical_geography') %>
+  <% public_body.get_tag_values('statistical_geography').each do |tag_value| %>
+      <%= link_to _('Explore local statistics'),
+                    "https://explore-local-statistics.beta.ons.gov.uk/areas/#{ tag_value }" %><br>
+  <% end %>
+<% end %>
```

Comment:
Our :robot: overlords have suggested that we can use a bit of recursion to handle cases where either tag (or both exists… 

```suggestion
<% ons_statistics_tags = [] %>
<% if public_body.has_tag?('statistical_geography') %>
  <% ons_statistics_tags += public_body.get_tag_values('statistical_geography') %>
<% end %>
<% if public_body.has_tag?('gss') %>
  <% ons_statistics_tags += public_body.get_tag_values('gss') %>
<% end %>

<% unless ons_statistics_tags.empty? %>
  <% ons_statistics_tags.each do |tag_value| %>
    <%= link_to _('Explore local statistics'),
          "https://explore-local-statistics.beta.ons.gov.uk/areas/#{ tag_value }" %><br>
  <% end %>
<% end %>
```

Reply:
This looks good to me.